grants/grant_application_template.md

```diff
@@ -30,26 +31,22 @@ Please describe the team's relevant experience
 * https://www.linkedin.com/<person_2>
 
 ## Development Roadmap
-Please add information setting into sufficient detail the following (usually two pages):
+This section should break out the development roadmap into a number of milestones. Please ensure this section is sufficently detailed. The length will vary depending on the project but approximately 2 pages is a good guideline. It should also be clear how the project is realted to Substrate and/or Polkadot.
 
-* The specifications of the software, delivered over a number of milestones. The detail must be such that after delivery we are able to examine whether the software meets the requirements agreed in the specifications, in order to be able to make any payments to you.
-* A time schedule on when each milestone will be achieved.
-* The required grant per milestone and the total amount requested. Grants can be in one of the fiat currencies CHF, EUR or USD or in DOTs or a combination of fiat and DOTs. Please reach out to grants@web3.foundation for discussing what amount in fiat and DOTs would be appropriate. 
-* Note that we now have a preference to collect the financial information via our Google Form, or via a Google Document (paste the link into our Google Form).
-
-
-Note: It should be clear how the project will be working towards deployment on Substrate / Polkadot.
-
-Also helpful to know: what are the team’s long-term plans and intentions after the grant has been awarded?
+Please provide the following:
+  * For each milestone:
+    * The specification of the software achieved after completion of each milestone. The level of detail must be high enough so that we are able to test that the software meets the specification.
+    * The total amount of funding requested per milestone. Funding can be in fiat (CHF, EUR or USD) or in DOTs. It can also be in a combination of fiat and DOTs. Please reach out to grants@web3.foundation to discuss what amount in fiat and DOTs would be appropriate for your project.
+  * A schedule indicating when each milestone will be achieved.
+  * The team's long-term plans and intentions 
 
+Note that we now have a preference to collect financial information (i.e. funding per milestone) via our Google Form, or via a Google Document link pasted into the link into our Google Form.
```

Comment:
"via our Google Form, or via a Google Document link pasted into the link into our Google Form."

This could be made more clear, not really sure what this sentence means.